db/migrate/20160907122352_add_more_foreign_keys_to_invoices.rb

```diff
@@ -0,0 +1,6 @@
+class AddMoreForeignKeysToInvoices < ActiveRecord::Migration[5.0]
+  def change
+    add_foreign_key :invoices, :merchants
+    add_foreign_key :invoices, :customers
+  end
+end
```

Comment:
I'm glad you caught this, too. I thought about doing this myself last night, but figured I'd wait until today in case there were going to be any merge conflicts.